helpers/image_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'tempfile'
+require_relative '../config/StaticData'
+
+# Class for working with images after convert service
+class ImageHelper
+  def self.get_image_size(url)
+    Tempfile.create('images-', StaticData::TMP_FOLDER.to_s) do |fo|
+      fo.binmode
+      fo.write(URI.parse(url).open(&:read))
+      fo.rewind
+      File.size(fo.path)
```

Comment:
I'm not sure if File.size is inside block of `Tempfile.create...` - this method return not nil or zero, but correct file size. Please recheck it

Reply:
I don't think that really matters, because Tempfile delegates to File.

Reply:
Agree with you, my bad